ixblue_c3_ins/include/ixblue_c3_ins/c3_protocol.h

```diff
@@ -1,13 +1,36 @@
-/*  QinetiQ North America (QNA)
- *  350 Second Avenue
- *  Waltham, MA 02451
+/*********************************************************************
+ * Software License Agreement (BSD License)
  *
- *  Proprietary Licensed Information.
- *  Not to be duplicated, used, or disclosed,
- *  except under terms of the license.
+ *  Copyright (c) 2018-2020, QinetiQ, Inc.
```

Comment:
nit: 2021

Reply:
Oops, good point. I'm fine with changing this here.

Reply:
See 6c4e7db70f748740cb8374f08e097e0c0ef1ea95

Reply:
Great. LGTM